### DISCUSSION TITLE
"Folders" won't work for non-techies

### DISCUSSION THREAD

**Discussion by aagha** [requarks/wiki#6680] [Help / Questions] [answered]
I've just played with wiki.js for a few hours after getting it to run on my locker Docker.

We're in the process of trying to setup a knowledge base for our company, and I was really enthusiastic about wiki.js--it is clean, has great configuration options, and looks simple to use.  The rub is this last point--trying to figure out how to create files, the paths where they should live, the structure, is just not something anyone but a technical person will be able to do.

That means, my business partner, who's non-technical and would be responsible for keeping the wiki documented and up to date with sales materials, meeting notes, customer docs, etc., would never be able to figure out where files go, under what structure/path, etc.

Has anyone figured out a simple way around this? A plugin that creates the illusion of "folders" out of paths?

**Comment by NGPixel** [comment, marked as answer]
This is already addressed in v3 which allows for traditional folders management.
